README.md

```diff
@@ -32,14 +32,14 @@ which has fundamental issues (like no comments).
 
 ## This solution
 
-Put all of your scripts in a file called `package-scripts.js` and use `p-s` in a single `package.json` script:
+Put all of your scripts in a file called `package-scripts.js` and use `nps` in a single `package.json` script:
```

Comment:
Let's mention that `nps` is the recommended alias right here. Otherwise people may be confused.